### PR TITLE
for better understanding

### PR DESCRIPTION
**Pull request by DiamondeX**

### CODE REVIEW

File: book/02-git-basics/sections/recording-changes.asc [hunk 1]
```diff
@@ -28,7 +28,7 @@ Your branch is up-to-date with 'origin/master'.
 nothing to commit, working directory clean
 ----
 
-This means you have a clean working directory – in other words, there are no tracked and modified files.
+This means you have a clean working directory – in other words, there are no modified files among tracked ones.
```
Comment (ben) [inline book/02-git-basics/sections/recording-changes.asc]: Hmm. What about this:

> This means you have a clean working directory – in other words, there are no tracked files that are modified.

Comment (DiamondeX) [inline book/02-git-basics/sections/recording-changes.asc]: That is good too.

Comment (412445773) [inline book/02-git-basics/sections/recording-changes.asc]: GOOD!

Comment (trych) [inline book/02-git-basics/sections/recording-changes.asc]: I actually thought the _tracked_ is a mistake altogether and the sentence is supposed to be:

> This means you have a clean working directory – in other words, there are no staged and modified files.

Which I would change to:
> This means you have a clean working directory – in other words, there are no modified or staged files.

If the tracked is indeed supposed to be there, I would simplify the sentence even more:

> This means you have a clean working directory – in other words, no tracked files are modified.

Comment (DiamondeX) [inline book/02-git-basics/sections/recording-changes.asc]: Well, guys, I think my duty was to draw your attention to this ambiguity and your duty is to eliminate it

**Comment by trych** [comment]
I find
> This means you have a clean working directory – in other words, none of your tracked files are modified.

the most clear version. Other opinions?

File: book/02-git-basics/sections/recording-changes.asc (hunk 1 above)
Comment (ben) [inline book/02-git-basics/sections/recording-changes.asc]: ✨ I love this:

> This means you have a clean working directory – in other words, none of your tracked files are modified.

@DiamondeX, can you push it to a new commit on this branch?

Comment (DiamondeX) [inline book/02-git-basics/sections/recording-changes.asc]: Perfectly!
...Looks like I done as you expected! ;)

**Comment by DiamondeX** [comment]
Made changes in accordance to this discussion: 
https://github.com/progit/progit2/pull/617/files/999ed4e6ce144d210f68d85847fb090d94986764#r74694810

**Comment by ben** [comment]
:sparkles:

**Pull request merged**